bundle/Core/Pagination/Pagerfanta/RelatedContentAdapter.php

```diff
@@ -78,6 +108,10 @@ public function getNbResults()
      * @param int $offset The offset
      * @param int $length The length
      *
+     * @throws \eZ\Publish\API\Repository\Exceptions\NotFoundException
```

Comment:
Also superfluous.

Reply:
Fixed in https://github.com/netgen/TagsBundle/pull/98/commits/f323b02f1c1dbcedffd2e2c09268ba3d07ac8f5f.